video/out/opengl/video.c

```diff
@@ -3006,13 +3006,13 @@ void gl_video_render_frame(struct gl_video *p, struct vo_frame *frame,
     if (!has_frame || !mp_rect_equals(&p->dst_rect, &target_rc)) {
         struct m_color c = p->clear_color;
         float color[4] = {c.r / 255.0, c.g / 255.0, c.b / 255.0, c.a / 255.0};
-        p->ra->fns->clear(p->ra, target.tex, color, &target_rc);
+        p->ra->fns->clear(p->ra, target.tex, color);
```

Comment:
How about this: clear() args stay as they are, but in vulkan the clear is just “remembered” for the next time that texture gets rendered to. If we end up hitting `ra_vk_present_frame` on a texture that still has a pending “clear”, it would insert a dummy renderpass.

Is it okay if we just remember the most recent clear()? I don't think we have a use case for clearing multiple subrects of a single image inside the same renderpass.